packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -782,12 +781,12 @@ contract VotingV2 is
     /**
      * @notice Resets the Gat percentage. Note: this change only applies to rounds that have not yet begun.
      * @dev This method is public because calldata structs are not currently supported by solidity.
-     * @param newGatPercentage sets the next round's Gat percentage.
+     * @param newGat sets the next round's Gat percentage.
      */
-    function setGatPercentage(uint256 newGatPercentage) public override onlyOwner {
-        require(newGatPercentage < 1e18);
-        gatPercentage = newGatPercentage;
-        emit GatPercentageChanged(newGatPercentage);
+    function setGat(uint256 newGat) public override onlyOwner {
+        require(newGat < votingToken.totalSupply() && newGat > 0);
```

Comment:
I wonder if we would want to set a higher minimum, for example 5% of the totalSupply, or leave it completely open to the decision of the token holders.

Reply:
totally open. I dont think we should restrict this.